warehouse/query-core/src/test/java/datawave/query/edge/ExtendedEdgeQueryLogicTest.java

```diff
@@ -69,6 +69,22 @@ public void testEdgeQuerySyntax_WithQueryModel() throws Exception {
         compareResults(logic, expected);
     }
     
+    @Test
+    public void testEdgeQuerySyntaxLuceneWithQueryModel() throws Exception {
```

Comment:
I like this test. I'd be interested in seeing a JEXL syntax query added in here as well for parity, just to see how the logic handles both syntaxes if that seems reasonable.

Reply:
since jexl is the default param, other test cases are essentially testing jexl syntax. I feel like adding another test to test that would be overkill. instead I think adding the param to one of the existing one should cover that.

Reply:
Sounds good to me, thanks